### PR TITLE
[Cleanup]: Fix Android Studio Warnings #13282

### PR DESCRIPTION
**Pull request by rishi2062**
## Purpose / Description
Getting Familiar with Project.

## Fixes
* #13282

 [AbstractFlashCardViewer]  - Kotlin 1.3 and above supports omission of paranthesis , if constructor is empty , 
[AnkiActivity] - CustomTabsIntent qualifier name is not needed externally, 
[NavigationDrawerActivity] - By default open classes are public in kotlin, 
[ReadText] - unused import directive and also (.*) can be avoided for clean code

## Approach
Removal of warnings, to maintain the clean code practice

### CODE REVIEW

**Review by david-allison — CHANGES_REQUESTED** [review]
Looks good! One requested change

File: AnkiDroid/src/main/java/com/ichi2/anki/AnkiActivity.kt [hunk 1]
```diff
@@ -331,7 +330,7 @@ open class AnkiActivity : AppCompatActivity, SimpleMessageDialogListener {
             .setToolbarColor(toolbarColor)
             .setNavigationBarColor(navBarColor)
             .build()
-        val builder = CustomTabsIntent.Builder(customTabActivityHelper.session)
+        val builder = Builder(customTabActivityHelper.session)
```
[the remark this answers is not in the document]
Comment (rishi2062) [reply AnkiDroid/src/main/java/com/ichi2/anki/AnkiActivity.kt:333]: Sure, reverting it.

**Review by david-allison — APPROVED** [review]
Cheers!

**Review by lukstbit — APPROVED** [review]
Thanks!

**Comment by david-allison** [comment]
@rishi2062 ping me on Discord for your roles if you'd like. Congrats on your first PR!

**Comment by rishi2062** [comment]
Thank you. 
yeah i will ping you.

**Pull request merged**